src/test/java/org/trebol/jpa/services/crud/UsersJpaCrudServiceTest.java

```diff
@@ -24,13 +28,15 @@
 class UsersJpaCrudServiceTest {
   @Mock IUsersJpaRepository usersRepositoryMock;
   @Mock ITwoWayConverterJpaService<UserPojo, User> usersConverterMock;
+  @Mock SecurityProperties securityPropertiesMock;
   private GenericCrudJpaService<UserPojo, User> instance;
 
   @BeforeEach
   void beforeEach() {
     instance = new UsersJpaCrudServiceImpl(
             usersRepositoryMock,
-            usersConverterMock
+            usersConverterMock,
+            securityPropertiesMock
     );
   }
```

Comment:
test: decorate `instance` with `@InjectMocks` instead of manually initializing it